tests/dat/actions/asyncError.js

```diff
@@ -0,0 +1,13 @@
+function main(params) {
+    var str = params.payload;
+
+    return whisk.invoke({
+        name: 'wc',
```

Comment:
i still think it's unnecessary to test with whisk.invoke because the issue was general to any failed promise but will acquiesce.